### PR TITLE
Adds the method getKVPairData()

### PR DESCRIPTION
**Pull request by KambleSonam**
- Adds the new method  getKVPairData() for Web custom KV pair
- Updated the Web SDK version to v1.9.0

### CODE REVIEW

**Review by darshanclevertap — APPROVED** [review]

File: lib/clevertap_plugin_web.dart [hunk 1]
```diff
@@ -5,13 +5,13 @@ import 'package:flutter/services.dart';
...
 import 'package:js/js_util.dart' as js_util;
 import 'package:js/js.dart';
-import 'dart:html' as html;
 
 /// A web implementation of the CleverTapPlugin plugin.
 class CleverTapPlugin {
   static MethodChannel? _nativeToDartMethodChannel;
 
   static void registerWith(Registrar registrar) {
+    print("registerWith");
```
Comment (Anush-Shand) [inline lib/clevertap_plugin_web.dart:14]: Was this only for internal testing purposes or does it need to be a part of the sdk?

**Review by Anush-Shand — APPROVED** [review]
LGTM

File: lib/clevertap_plugin_web.dart (hunk 1 above)
Comment (KambleSonam) [reply lib/clevertap_plugin_web.dart:14]: That was for internal testing

Comment (Anush-Shand) [reply lib/clevertap_plugin_web.dart:14]: Can be removed then?

Comment (KambleSonam) [reply lib/clevertap_plugin_web.dart]: Done.

**Pull request merged**